torch_np/_binary_ufuncs.py

```diff
@@ -57,3 +57,58 @@ def wrapped(
     decorated.__qualname__ = name  # XXX: is this really correct?
     decorated.__name__ = name
     vars()[name] = decorated
+
+
+# a stub implementation of divmod, should be improved after
+# https://github.com/pytorch/pytorch/issues/90820 is fixed in pytorch
+#
+# Implementation details: we just call two ufuncs which have been created
+# just above, for x1 // x2 and x1 % x2.
+# This means we are normalizing x1, x2 in each of the ufuncs --- note that there
+# is no @normalizer on divmod.
+
+
+def divmod(
+    x1,
+    x2,
+    /,
+    out=None,
+    *,
+    where=True,
+    casting="same_kind",
+    order="K",
+    dtype=None,
+    subok: SubokLike = False,
+    signature=None,
+    extobj=None,
+):
+    out1, out2 = None, None
+    if out is not None:
+        out1, out2 = out
+
+    kwds = dict(
+        where=where,
+        casting=casting,
+        order=order,
+        dtype=dtype,
+        subok=subok,
+        signature=signature,
+        extobj=extobj,
+    )
+
+    # NB: use local names for
+    quot = floor_divide(x1, x2, out=out1, **kwds)
+    rem = remainder(x1, x2, out=out2, **kwds)
```

Comment:
Note that here (and in general) if we want these functions to be differentiable in PyTorch, we should not use their `out=` variant. We should implement the `out=` behaviour manually.

Reply:
Here these are wrapper `floor_divide` and `remainder`, so `torch.remainder` and `torch.floor_divide` never see the `out=`. 
Also NB: this will need a rework anyway, apparently there are more (out1, out2) ufuncs, ldexp and frexp. So `Tuple[NDArray]` return annotation will need to appear after we settle on the generic machinery.